awx/ui_next/src/screens/Inventory/InventorySources/InventorySourceList.jsx

```diff
@@ -49,8 +50,24 @@ function InventorySourceList({ i18n }) {
     {
       sources: [],
       sourceCount: 0,
+      sourceChoices: [],
     }
   );
+  const canSyncSources =
+    sources.length > 0 &&
+    sources.every(source => source.summary_fields.user_capabilities.start);
+  const {
+    isLoading: isSyncAllLoading,
+    error: syncAllError,
+    request: syncAll,
+  } = useRequest(
+    useCallback(async () => {
+      if (canSyncSources) {
+        await InventoriesAPI.syncAllSources(id);
+        fetchSources();
```

Comment:
Do we need to fetch a new list after every sync? The old UI is a little different because it receives status updates over web sockets, but we don't fetch a new source list when the sync resolves.

Reply:
Resolved offline